packages/react-native-codegen/src/parsers/typescript/parser.js

```diff
@@ -95,7 +102,22 @@ class TypeScriptParser implements Parser {
   parseFile(filename: string): SchemaType {
```

Comment:
Also, all these methods looks practically identical one to each other, we may also think of replace the Parser interface with a `BaseParser` and share that code.